src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -74,6 +92,7 @@ const JsonLocalizer = () => {
             .filter(Boolean)
             .map(([key, values]) => {
               const cellWidth = 100 / values.length;
+              const showTextArea = activeCellId?.split('-')[0] === key;
```

Comment:
We use the same value in at least three places `activeCellId?.split('-')[0]`. Can we keep it in state?

Reply:
edited